sdcDataGeneration/primaryDataGeneration.js

```diff
@@ -40,26 +42,26 @@ for (let j = 0; j < 10000; j++) {
   });
 
   //place target filename in fs.createWriteStream
-  writer.pipe(fs.createWriteStream('./sdcDataGeneration/primaryentries17.csv', {flags: 'a'}));
+  writer.pipe(fs.createWriteStream('./sdcDataGeneration/primaryentries19.csv', {flags: 'a'}));
 
   for (let i = 1; i <= 1000; i++) {
 
     let id = i + (j * 1000);
     let photoid = i % 1000 + 1;
-    let zestimationPricea = Math.floor(Math.random() * 4500000) + 500000;
-    let thirtyDayPriceChangea = faker.random.number({'min': 15000, 'max': 50000});
-    let oneYearForcasta = faker.random.number({'min': 500000, 'max': (500000 + 100000)});
-    let comparableHomePricea = faker.random.number({'min': (500000 - 100000), 'max': (500000 + 100000)});
-    let marketAppreciationPricea = faker.random.number({'min': (500000 - 200000), 'max': (500000)});
+    let zestimationPricea = (Math.floor(Math.random() * 4500000) + 500000).toLocaleString();
+    let thirtyDayPriceChangea = faker.random.number({'min': 15000, 'max': 50000}).toLocaleString();
```

Comment:
I would recommend replacing faker.random.number with Math.number wherever possible for efficiency gains (~30% faster)